File: ApiSuvesaPos/Negocio/Modelo/cxp/devoluciones_compras/devoluciones_compras.vb
Namespace Modelo
    Public Class devoluciones_compras
        Public Property Devolucion As Long
        Public Property Id_Factura_Compra As Long
        Public Property SaldoAnt_Fact As Decimal
        Public Property SubTotalGravado As Decimal
        Public Property SubTotalExcento As Decimal
        Public Property Descuento As Decimal
        Public Property Impuesto As Decimal
        Public Property Monto As Decimal
        Public Property Fecha As DateTime
        Public Property Contabilizado As Boolean
        Public Property Asiento As Decimal
        Public Property Anulado As Boolean
        Public Property Cedula_Usuario As String
        Public Property Cod_Moneda As Integer
        Public Property NombrePro As String
        Public Property ContaInventario As Boolean
        Public Property AsientoInventario As Decimal
        Public Property FechaEntrada As DateTime

    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxp/devoluciones_compras/devoluciones_compras.cs
using System;

namespace Modelo
{
    public class devoluciones_compras
    {
        public long Devolucion { get; set; }
        public long Id_Factura_Compra { get; set; }
        public decimal SaldoAnt_Fact { get; set; }
        public decimal SubTotalGravado { get; set; }
        public decimal SubTotalExcento { get; set; }
        public decimal Descuento { get; set; }
        public decimal Impuesto { get; set; }
        public decimal Monto { get; set; }
        public DateTime Fecha { get; set; }
        public bool Contabilizado { get; set; }
        public decimal Asiento { get; set; }
        public bool Anulado { get; set; }
        public string Cedula_Usuario { get; set; }
        public int Cod_Moneda { get; set; }
        public string NombrePro { get; set; }
        public bool ContaInventario { get; set; }
        public decimal AsientoInventario { get; set; }
        public DateTime FechaEntrada { get; set; }
    }
}
